rules-reviewed/jboss-eap/websphere/WebSphereMQ.windup.xml

```diff
@@ -0,0 +1,1363 @@
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeConnectionFactory"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQe API." description="A more general, and recommended, approach is to use the ConnectionFactory class.">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/jms/package-summary.html"></link>
+                </classification>
+                <hint effort="1">
+                    <message>Replace with Java EE 6 JMS: `javax.jms.ConnectionFactory`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mqe-14wscy">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeJNDIConnectionFactory"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQe API." description="A more general, and recommended, approach is to use the ConnectionFactory class.">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/jms/package-summary.html"></link>
+                </classification>
+                <hint effort="1">
+                    <message>Replace with Java EE 6 JMS: `javax.jms.ConnectionFactory`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mqe-16aynk">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeJNDIQueueConnectionFactory"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQe API." description="A more general, and recommended, approach is to use the ConnectionFactory class.">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/jms/package-summary.html"></link>
+                </classification>
+                <hint effort="1">
+                    <message>Replace with Java EE 6 JMS: `javax.jms.ConnectionFactory`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mqe-18bicn">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeQueueConnectionFactory"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQe API." description="A more general, and recommended, approach is to use the QueueConnectionFactory class.">
```

Comment:
These classifications should not end with a period ".". None of our other classifications do this. Simple fix, just remove the period.